src/js/bellows.js

```diff
@@ -176,7 +176,7 @@
         close: function($item) {
             $item = this._item($item);
 
-            if (!$item.hasClass(cssClasses.OPENED)) {
+            if (!$item.hasClass(cssClasses.OPENED) || $item.hasClass(cssClasses.DISABLED)) {
```

Comment:
Totally agree.
Would something like `if (!_isOpen || _isDisabled)` be good enough? Or would we need something like `if (_isClosed || _isDisabled)`?